app/commands/decidim/verifications/authorize_user.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Verifications
+    # A command to authorize a user with an authorization handler.
+    class AuthorizeUser < Decidim::Command
+      # Public: Initializes the command.
+      #
+      # handler - An AuthorizationHandler object.
+      def initialize(handler, organization)
+        @handler = handler
+        @organization = organization
+      end
+
+      # Executes the command. Broadcasts these events:
+      #
+      # - :ok when everything is valid.
+      # - :invalid if the handler wasn't valid and we couldn't proceed.
+      #
+      # Returns nothing.
+      def call
+        if handler.invalid?
+          conflict = create_verification_conflict
+          notify_admins(conflict) if conflict.present?
+
+          return broadcast(:invalid)
+        end
+
+        Authorization.create_or_update_from(handler)
+
+        broadcast(:ok)
+      end
+
+      private
+
+      attr_reader :handler
+
+      def notify_admins(conflict)
+        Decidim::EventsManager.publish(
+          event: "decidim.events.verifications.managed_user_error_event",
+          event_class: Decidim::Verifications::ManagedUserErrorEvent,
+          resource: conflict,
+          affected_users: Decidim::User.where(admin: true, organization: @organization)
```

Comment:
Hi, is it possible to fetch admins directly from the organization to use cache ? 
```suggestion
          affected_users: @organization.admins
```